sasmodels/bumps_model.py

```diff
@@ -66,8 +66,9 @@ def create_parameters(model_info, **kwargs):
         value = kwargs.pop(p.name, p.default)
         pars[p.name] = BumpsParameter.default(value, name=p.name, limits=p.limits)
         if p.polydisperse:
+            pd_limits = (-360.0, 360.0) if p.type == "orientation" else (0., 1.)
```

Comment:
Correct me if I'm wrong, but doesn't this now override any value entered by the user.